vmux/__main__.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#! /usr/bin/env python3
```

Comment:
The space shouldn't be there.

Reply:
The space after a hashbang is optional, but I'll revert it.